pkg/utils/errors.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+func JoinErrors(errors []error) error {
```

Comment:
This is more of a comment about our whole code base and not this PR... I think we should eventually switch to using the stock golang errors package and use wrapping(%w)/errors.Unwrap wherever necessary/reasonable. If/When we do that, maybe we should also take a look at  [multierror](https://github.com/hashicorp/go-multierror) for handling multiple errors.